src/javascript/Edit/Edit.actions.js

```diff
@@ -3,7 +3,7 @@ import saveAction from './save/save.action';
 import publishAction from './publish/publish.action';
 import startWorkflow from './startWorkflow/startWorkflow.action';
 import copyLanguageAction from './copyLanguage/copyLanguage.action';
-import {Save, CloudUpload} from '@material-ui/icons';
+import {CloudUpload, Save} from '@material-ui/icons';
```

Comment:
You can use the moonstone icons instead of material-ui

Reply:
This change is only a swap of the values (to be in alphabetical order) not to change the display of the application, please create a ticket to handle that change that cannot be part of that PR (not same scope)